test/elm/interval/test.coffee

```diff
@@ -1405,3 +1405,699 @@ describe 'Collapse', ->
+    prettyList(@secPrecPerWeek.exec(@ctx)).should.equal e
+    
+    # define SecPrecPerDay: expand { Interval[@2018-01-01T01:00:00+00:00, @2018-01-03T01:00:00+00:00] } per day
+    e = "{ [2018-01-01T00:00:00+00:00, 2018-01-01T23:59:59+00:00], [2018-01-02T00:00:00+00:00, 2018-01-02T23:59:59+00:00] }"
+    prettyList(@secPrecPerDay.exec(@ctx)).should.equal e
+    
+    # define SecPrecPerHour: expand { Interval[@2018-01-01T01:00:00+00:00, @2018-01-01T03:00:00+00:00] } per hour
+    e = "{ [2018-01-01T01:00:00+00:00, 2018-01-01T01:59:59+00:00], [2018-01-01T02:00:00+00:00, 2018-01-01T02:59:59+00:00] }"
+    prettyList(@secPrecPerHour.exec(@ctx)).should.equal e
+    
+    # define SecPrecPerMinute: expand { Interval[@2018-01-01T01:00:00+00:00, @2018-01-01T01:02:00+00:00] } per minute
+    e = "{ [2018-01-01T01:00:00+00:00, 2018-01-01T01:00:59+00:00], [2018-01-01T01:01:00+00:00, 2018-01-01T01:01:59+00:00] }"
+    prettyList(@secPrecPerMinute.exec(@ctx)).should.equal e
+    
+    # define SecPrecPerSecond: expand { Interval[@2018-01-01T01:00:00+00:00, @2018-01-01T01:00:01+00:00] } per second
+    e = "{ [2018-01-01T01:00:00+00:00, 2018-01-01T01:00:00+00:00], [2018-01-01T01:00:01+00:00, 2018-01-01T01:00:01+00:00] }"
+    prettyList(@secPrecPerSecond.exec(@ctx)).should.equal e
+
+    should.not.exist @secPrecPerMillisecond.exec(@ctx)
+  
+  it 'expands a minute precision datetime', ->
+    # define MinPrecPerYear: expand { Interval[@2016-01-01T01:00+00:00, @2018-01-01T01:00+00:00] } per year
+    e = "{ [2016-01-01T00:00+00:00, 2016-12-31T23:59+00:00], [2017-01-01T00:00+00:00, 2017-12-31T23:59+00:00] }"
+    prettyList(@minPrecPerYear.exec(@ctx)).should.equal e
+
+    # define MinPrecPerMonth: expand { Interval[@2018-01-01T01:00+00:00, @2018-03-01T01:00+00:00] } per month
+    e = "{ [2018-01-01T00:00+00:00, 2018-01-31T23:59+00:00], [2018-02-01T00:00+00:00, 2018-02-28T23:59+00:00] }"
+    prettyList(@minPrecPerMonth.exec(@ctx)).should.equal e
+
+    # define MinPrecPerWeek: expand { Interval[@2018-01-01T01:00+00:00, @2018-01-21T01:00+00:00] } per week
+    e = "{ [2018-01-01T00:00+00:00, 2018-01-07T23:59+00:00], [2018-01-08T00:00+00:00, 2018-01-14T23:59+00:00] }"
+    prettyList(@minPrecPerWeek.exec(@ctx)).should.equal e
+    
+    # define MinPrecPerDay: expand { Interval[@2018-01-01T01:00+00:00, @2018-01-03T01:00+00:00] } per day
+    e = "{ [2018-01-01T00:00+00:00, 2018-01-01T23:59+00:00], [2018-01-02T00:00+00:00, 2018-01-02T23:59+00:00] }"
+    prettyList(@minPrecPerDay.exec(@ctx)).should.equal e
+    
+    # define MinPrecPerHour: expand { Interval[@2018-01-01T01:00+00:00, @2018-01-01T03:00+00:00] } per hour
+    e = "{ [2018-01-01T01:00+00:00, 2018-01-01T01:59+00:00], [2018-01-01T02:00+00:00, 2018-01-01T02:59+00:00] }"
+    prettyList(@minPrecPerHour.exec(@ctx)).should.equal e
+    
+    # define MinPrecPerMinute: expand { Interval[@2018-01-01T01:00+00:00, @2018-01-01T01:02+00:00] } per minute
```

Comment:
Another case where the CQL in the comment doesn't match the CQL in the fixture, so it makes the expected result look wrong to the reviewer, even when it is correct.